src/Config_Command.php

```diff
@@ -619,6 +619,54 @@ public function has( $args, $assoc_args ) {
 		}
 	}
 
+	/**
+	 * Refreshes the salts defined in the wp-config.php file
+	 *
+	 * ## OPTIONS
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Get new salts for your wp-config.php file
+	 *     $ wp config shuffle-salts
+	 *
+	 * @subcommand shuffle-salts
+	 * @when before_wp_load
+	 */
+	public function shuffle_salts( $args, $assoc_args ) {
+
+		try {
+			for ( $i = 0; $i < 8; $i++ ) {
+				$secret_keys[] = self::unique_key();
+			}
+		} catch ( Exception $ex ) {
+
+			$secret_keys = self::_read( 'https://api.wordpress.org/secret-key/1.1/salt/' );
+			$secret_keys = explode( "\n", $secret_keys );
+			foreach ( $secret_keys as $k => $v ) {
+				$secret_keys[$k] = substr( $v, 28, 64 );
+			}
+
+		}
+
+		$constant_list = array( 'AUTH_KEY', 'SECURE_AUTH_KEY', 'LOGGED_IN_KEY', 'NONCE_KEY', 'AUTH_SALT', 'SECURE_AUTH_SALT', 'LOGGED_IN_SALT', 'NONCE_SALT' );
+		$key = 0;
+
+		$path = $this->get_config_path();
+
+		try {
+			$config_transformer = new WPConfigTransformer( $path );
+			foreach ( $constant_list as $constant ) {
+				$config_transformer->update( 'constant', $constant, $secret_keys[ $key ] );
```

Comment:
Use `$secret_keys[ $constant ]` here instead. You can get rid of the numeric `$key` completely.